yajuH/taittirIyam/sUtram/ApastambaH/dharma-sUtram/haradatta-TIkA/2/08/19/11_sthAlIpAkAnudeshyAni_cha_varjayet.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "11 स्थालीपाकानुदेश्यानि च वर्जयेत्"
```

Comment:
केन कारणेन क्रमाङ्कपरिवर्तनम्?

स्थालीपकानुदेश्यानि च वर्जयेत् १५  इति सूत्रसङ्ख्या भिन्ना दृश्यते।

Reply:
## सूत्रम्
यावद्ग्रासं सन्नयन्नस्कन्दयन्नाऽपजिहीताऽपजिहीत वा कृत्स्नं ग्रासं ग्रसति सहाङ्गुष्ठम् ॥ ५॥  

श्रीमन् अस्य सूत्रस्य भागचतुष्टयम् आसीत्। नाम  चत्वारि सूत्राणि। पुस्तके तु एकमेव वर्तते इदं पूर्णं सूत्रम्। तस्मात् यथापुस्तकं क्रमाङ्कः परिवर्तितः।

Reply:
https://www.sanskritworld.in/public/assets/book/book_50fd6303f1675.txt इत्य् अनुसृत्यैव सूत्रसङ्ख्या ऽस्मत्सञ्चिकासु वर्तेत। नान्यथा। सङ्गृह्य हरदत्तेन व्याख्यातं चेद् - अन्तिमायां सूत्रसञ्चिकायां व्याख्यानं स्थापयित्वा, शिष्टासु सञ्चिकासु - "(अग्रे व्याख्यातम्)" इति लेख्यम्।

Reply:
oh तथा। अस्तु श्रीमन्, संशोधयामि

Reply:
संशोधितं श्रीमन्